### ISSUE TITLE
Undefined property: stdClass::$fg2

### ISSUE THREAD

**Issue by vaughany** [marxjohnson/moodle-block_accessibility#68]
We've just installed and pushed out the latest Accessibility block code from the 'dev' branch and noticed that the 'A' buttons lack colour. Looking at the Apache error log, I can see the following repeating errors (sanitised somewhat):

```
PHP Notice:  Undefined property: stdClass::$fg2 in blocks/accessibility/userstyles.php on line 146, referer: http://moodle.ac.uk/my/
PHP Stack trace:, referer: http://moodle.ac.uk/my/
PHP   1. {main}() blocks/accessibility/userstyles.php:0, referer: http://moodle.ac.uk/my/

PHP Notice:  Undefined property: stdClass::$bg2 in blocks/accessibility/userstyles.php on line 147, referer: http://moodle.ac.uk/my/
PHP Stack trace:, referer: http://moodle.ac.uk/my/
PHP   1. {main}() blocks/accessibility/userstyles.php:0, referer: http://moodle.ac.uk/my/

PHP Notice:  Undefined property: stdClass::$fg3 in blocks/accessibility/userstyles.php on line 146, referer: http://moodle.ac.uk/my/
PHP Stack trace:, referer: http://moodle.ac.uk/my/
PHP   1. {main}() blocks/accessibility/userstyles.php:0, referer: http://moodle.ac.uk/my/

PHP Notice:  Undefined property: stdClass::$bg3 in blocks/accessibility/userstyles.php on line 147, referer: http://moodle.ac.uk/my/
PHP Stack trace:, referer: http://moodle.ac.uk/my/
PHP   1. {main}() blocks/accessibility/userstyles.php:0, referer: http://moodle.ac.uk/my/

PHP Notice:  Undefined property: stdClass::$fg4 in blocks/accessibility/userstyles.php on line 146, referer: http://moodle.ac.uk/my/
PHP Stack trace:, referer: http://moodle.ac.uk/my/
PHP   1. {main}() blocks/accessibility/userstyles.php:0, referer: http://moodle.ac.uk/my/

PHP Notice:  Undefined property: stdClass::$bg4 in blocks/accessibility/userstyles.php on line 147, referer: http://moodle.ac.uk/my/
PHP Stack trace:, referer: http://moodle.ac.uk/my/
PHP   1. {main}() blocks/accessibility/userstyles.php:0, referer: http://moodle.ac.uk/my/
```
We're using block version 2.2 (version 2014072501) in Moodle 2.7.1+ (version 20140821).  I will look into this myself shortly but with the start of the new term looming, I may not get very far.

**Comment by marxjohnson** [comment]
Closing as this wasn't reproduced and no further details given.
